wpimath/src/main/native/include/frc/geometry/Translation3d.h

```diff
@@ -244,6 +248,32 @@ class WPILIB_DLLEXPORT Translation3d {
            units::math::abs(m_z - other.m_z) < 1E-9_m;
   }
 
+  /**
+   * Returns the nearest Translation3d from a collection of translations
+   * @param translations The collection of translations.
+   * @return The nearest Translation3d from the collection.
+   */
+  constexpr Translation3d Nearest(
+      std::span<const Translation3d> translations) const {
+    return *std::min_element(translations.begin(), translations.end(),
+                             [this](Translation3d a, Translation3d b) {
```

Comment:
Translation2d should be updated similarly, now that I'm looking at it. The rule of thumb is to pass anything larger than 64 bits by const ref.
```suggestion
                             [this](const Translation3d& a, const Translation3d& b) {
```

Reply:
Separate branch/PR.